src/Core/Layer/Layer.js

```diff
@@ -71,6 +71,8 @@ function GeometryLayer(id, object3d) {
         value: id,
         writable: false,
     });
+
+    this.postUpdate = () => {};
```

Comment:
The `preUpdate` is mandatory while the `postUpdate` isn't.

IMHO implementing default optionnal methods eases readability of the `MainLoop` but if you prefer I can revert to `if (layer.postMethod) layer.postMethod` style.